docs/SOPs/archiving_SOP.md

```diff
@@ -194,7 +198,7 @@ As a standard, we run bam conversion in the EBI cluster, as it may need a big ch
 
 1. Login to cluster
    ```
-   ssh noah-login
+   ssh ebi-cli.ebi.ac.uk
```

Comment:
I think both are valid, is there any difference between the 2?

Reply:
I sometimes encounter issues when using noah-login. So i consistently use ebi-cli.ebi.ac.uk.